aten/src/ATen/native/cudnn/Conv_v7.cpp

```diff
@@ -821,6 +821,109 @@ void raw_cudnn_convolution_backward_weight_out(
   TORCH_INTERNAL_ASSERT(false, "This case should not be dispatched to cuDNN.");
 }
 
+void raw_cudnn_convolution_add_relu_out(
+    const Tensor& output,
+    const Tensor& input,
+    const Tensor& weight,
+    const Tensor& bias,
+    const Tensor& z,
+    float alpha,
+    IntArrayRef padding,
+    IntArrayRef stride,
+    IntArrayRef dilation,
+    int64_t groups) {
+  // This function is currently only called if frozen-model opt is turned on.
+  // The benchmarking and caching mechanism is not ready, so let's set benchmark
+  // to false.
```

Comment:
https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnActivationMode_t : ` cudnnConvolutionBiasActivationForward() function must use CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_​PRECOMP_GEMM` - i'm not sure we can implement benchmarking at all.

Reply:
"Only the CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_​PRECOMP_GEMM algo is enabled with CUDNN_ACTIVATION_IDENTITY. In other words, in the cudnnActivationDescriptor_t structure of the input activationDesc, if the mode of the cudnnActivationMode_t field is set to the enum value CUDNN_ACTIVATION_IDENTITY, then the input cudnnConvolutionFwdAlgo_t of this function cudnnConvolutionBiasActivationForward() must be set to the enum value CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_​PRECOMP_GEMM. For more information, see cudnnSetActivationDescriptor()."  -- My interpretation is we can still do benchmarking if the activation function is RELU.